index.js

```diff
@@ -98,12 +97,20 @@ server.post("/cadastro", (request, response) => {
   }
   const owner = request.cookies.user;
 
+  var options = {     year: 'numeric',
+  month: ('long' || 'short' || 'numeric'),
+  weekday: ('long' || 'short'),
+  day: 'numeric',
+  hour: 'numeric',
+  minute: 'numeric',
+  second: 'numeric',};
+
   const database = JSON.parse(jetpack.read('./database.json'));
   const databaseArticle = database.article[formArticle.url] || {};
 
   database.article[formArticle.url] = {
     owner,
-    date: new Date().toJSON(),
+    date: new Date().toLocaleDateString('pt-br', options),
```

Comment:
Sempre evitamos salvar a data já formatada no banco pois fica complicado trabalhar com ela depois disso, prefira salvar como JSON mesmo.

```suggestion
    date: new Date().toJSON(),
```